src/components/ObsControl.jsx

```diff
@@ -25,13 +25,17 @@ export class ObsControl extends addMoreDecorator(Component) {
     this.setAbnormal = this.setAbnormal.bind(this);
   }
 
-  onValueChangeDone() {
-    if (this.props.onEventTrigger) {
+  isAllowedToTriggerControlEvent(triggerControlEvent) {
+    return triggerControlEvent === undefined || triggerControlEvent;
```

Comment:
@vinay-ThoughtWorks for what **triggerControlEvent** value "isAllowedToTriggerControlEvent()" will return false ??

Reply:
When triggerControlEvent is false.

Reply:
We do not want the onValueChangeDone to get called on every call of onChange, it should be called if the parent wants to trigger the control event.